backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java

```diff
@@ -130,6 +130,15 @@ public static <T extends GeneratedMessageV3> StatusRuntimeException logError(
                 .setCode(modelDBException.getCode().value())
                 .setMessage(modelDBException.getMessage())
                 .build();
+      } else if (e instanceof IllegalArgumentException) {
+        var illegalArgumentException = (IllegalArgumentException) e;
+        logBasedOnTheErrorCode(
+            isClientError(Code.INVALID_ARGUMENT_VALUE), illegalArgumentException);
+        status =
+            Status.newBuilder()
+                .setCode(Code.INVALID_ARGUMENT_VALUE)
+                .setMessage(illegalArgumentException.getMessage())
+                .build();
```

Comment:
illegalArgumentException will thrown due to some internal mistakes not due to client's invalid argument i think. NO?

Reply:
In the modeldb and registry backend i am sure i do not used it anywhere for client parameter validation.

Reply:
For example: https://github.com/VertaAI/modeldb/blob/553e74086a34ce65254b8f0e89dea1c16b1a902d/backend/src/main/java/ai/verta/modeldb/project/FutureProjectDAO.java#L1735

Reply:
That doesn't look like internal error code, more like invalid argument

Reply:
Yeah got it. Ok, then we can add it.